mmv1/third_party/terraform/website/docs/d/compute_networks.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+subcategory: "Compute Engine"
+description: |-
+  List networks in a Google Cloud project.
+---
+
+# google\_compute\_networks
+
+List all networks in a specified Google Cloud project.
+
+## Example Usage
+
+```tf
+data "google_compute_networks" "my-networks" {
+  project = "my-cloud-project"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Required) The name of the project.
```

Comment:
Should `project` be required or optional? In the file `mmv1/third_party/terraform/services/compute/data_source_google_compute_networks.go`, `project` is optional.

Reply:
I created the doc before I made project optional.